src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -112,7 +117,8 @@ class PaymentMethodList extends Component {
     getFilteredPaymentMethods() {
         // Hide any billing cards that are not P2P debit cards for now because you cannot make them your default method, or delete them
         const filteredCardList = _.filter(this.props.cardList, card => card.accountData.additionalData.isP2PDebitCard);
-        let combinedPaymentMethods = PaymentUtils.formatPaymentMethods(this.props.bankAccountList, filteredCardList, this.props.payPalMeData);
+        const pendingBankAccount = PaymentUtils.getPendingBankAccount(this.props.personalBankAccount, this.props.plaidData);
```

Comment:
```suggestion
        const pendingBankAccount = (this.props.personalBankAccount && this.props.plaidData) ? PaymentUtils.getPendingBankAccount(this.props.personalBankAccount, this.props.plaidData) : {};
```

Reply:
Do you think this is worth it? 

A lot of our methods will run on the assumption that something may be empty no? `_.each(bankAccountList, (bankAccount) => {` can still run if the Onyx key bankAccountlist returned empty

Reply:
We can leave this as it is since there's an [early return](https://github.com/Expensify/App/blob/147e723354113ef2621b849572df67b205660344/src/libs/PaymentUtils.js#L107-L110) in getPendingBankAccount 👍🏼